docs/next.config.js

```diff
@@ -155,7 +156,8 @@ module.exports = {
     }
 
     // We want to speed-up the build of pull requests.
-    if (process.env.PULL_REQUEST === 'true') {
+    // TODO: revert. Only flipped for debugging
+    if (process.env.PULL_REQUEST === 'false') {
```

Comment:
```suggestion
    if (process.env.PULL_REQUEST === 'true') {
```
Apply this before merge.